HW/baseball.sql

```diff
@@ -0,0 +1,29 @@
+
+-- first join the game and batter counts table and save the results as a temporary table
+drop temporary table if exists joined_batter_game;
+create temporary table joined_batter_game
+select game.local_date,batter_counts.batter,batter_counts.game_id,batter_counts.Hit,batter_counts.atBat from batter_counts join game on batter_counts.game_id=game.game_id;
+-- change batter data type to character because it was integer
+alter table joined_batter_game modify batter varchar(100);
+-- calculate annual batting average
+drop temporary table if exists annual_at_bat_per_player;
```

Comment:
You might not want to drop the table right away. I think we're using these for future assignments.

Reply:
@seanfahey1 why should I drop the table right away?